notification_lib/novu_manager/generic_novu_manager.py

```diff
@@ -77,9 +81,20 @@ def format_steps_to_create_update(
                 "content": step["template"]["content"],
             }
             if step["template"]["type"] == "push":
+                # Push notification has "title" as their content key
                 formated_template["title"] = step["template"]["title"]
-            if step["template"]["type"] == "email":
+
+            # If customHtml is true, it means we are passing a custom html
+            if (
+                step["template"]["type"] == "email"
+                and step["template"]["contentType"] == "customHtml"
+            ):
+                # Subject is the title of the email
```

Comment:
what if email has not customHtml ? the template will not be created in this case